docs/docs/path_algorithm.md

```diff
@@ -0,0 +1,406 @@
+---
```

Comment:
With regard to `docs/docs/images/graph_query_city.png`
I'm not sure the picture needs to included the query, if the query is necessary consider adding it as text
Also consider using different color gray seems dull, lastly if possible use different colors to distinguish nodes from edges.

Reply:
You cannot control the colors in RI. If there is only one node type it would always be grey. Same for relationships.